bcda/models/service_test.go

```diff
@@ -445,24 +445,36 @@ func (s *ServiceTestSuite) TestGetQueJobs() {
 	}
 
 	since := time.Now()
+	terminationDetails := &Termination{
+		AttributionStrategy: AttributionHistorical,
+		TerminationDate:     time.Now().Add(-30 * 24 * time.Hour).Round(time.Millisecond).UTC(),
+	}
+
+	sinceAfterTermination := terminationDetails.TerminationDate.Add(10 * 24 * time.Hour).Round(time.Millisecond).UTC()
```

Comment:
Nit - we shouldn't need the Round/UTC call here since we've already performed that on the TerminationDate.

Reply:
ah, good point!